src/modules/data/entities/instance/instance.facilitycontrol.entity.ts

```diff
@@ -8,46 +10,55 @@ import {Faction, factionArray} from '../../constants/faction.consts';
 @Index(['instance', 'facility', 'timestamp'], {unique: true})
 export default class InstanceFacilityControlEntity {
     @ObjectIdColumn()
+    @Exclude()
     _id: ObjectID;
 
+    @ApiProperty({example: '10-12345', description: 'The Server-CensusInstanceId combination'})
     @Column({
         type: 'string',
     })
     instance: string;
 
+    @ApiProperty({example: 222, description: 'Facility ID'})
     @Column({
         type: 'number',
     })
     facility: number;
 
+    @ApiProperty({example: new Date(), description: 'Time of event instance in UTC'})
     @Column({
         type: 'date',
     })
     timestamp: Date;
 
+    @ApiProperty({enum: factionArray, description: 'Losing faction that lost control of the facility'})
     @Column({
         type: 'enum',
         enum: factionArray,
     })
     oldFaction: Faction;
 
+    @ApiProperty({enum: factionArray, description: 'Winning faction that took control of the facility'})
     @Column({
         type: 'enum',
         enum: factionArray,
     })
     newFaction: Faction;
 
+    @ApiProperty({example: 0, description: 'Time taken to capture/defend the facility'})
```

Comment:
Slightly wrong, this is time since the last capture / defense event, which is in seconds